gdeployfeatures/ctdb/ctdb.py

```diff
@@ -66,6 +67,14 @@ def ctdb_setup(section_dict):
             ctdbnodes = [ctdbnodes]
         section_dict['ctdbnodes'] = "\n".join(ctdbnodes)
         Global.logger.info("Using %s as CTDB nodes" % ctdbnodes)
+    else:
+        # Ensure that the section_dict['nodes'] are IPs and
+        # not hostnames when ctdb_nodes are not specified
+        for s in Global.hosts:
+            if s.isalpha() is True:
```

Comment:
if s.isalpha():
     ...

should be enough. 

Instead of `s' why not say for host in Global.hosts ... more readable. What does s stand for?